package-system/Lua/build_config.json

```diff
@@ -68,30 +56,17 @@
                "-DCMAKE_OSX_ARCHITECTURES=arm64",
                "-DCMAKE_THREAD_LIBS_INIT=-pthread",
                "-DCMAKE_C_FLAGS=\"-fPIC -O2 -Wall -Wextra -DLUA_COMPAT_5_2\""
```

Comment:
Don't we need a DEPLOYMENT_TARGET here too?  The other packages use the toollchain file to define that.   I see you ahve 10.2 for the MAC version, isn't there somethign similar for IOS?

Reply:
Possibly, I made updates to this PR so I can pull and run these scripts on Mac. I will find out if there are any issues when I build the iOS libs

Reply:
I added OSX_DEPLOYMENT_TARGET here anyways, although it didn't affect my particular build because my Mac already meets the minimum that I set. I set it to the latest version that I see set for iOS in all of the other build existing 3P scripts.